src/test/resources/schema/segment_same/Base_Structure.gql

```diff
@@ -427,8 +430,6 @@ JavascriptMemberExpressionArtifact sub JavascriptSourceArtifact,
 	plays has_javascript_property_role;
 
 JavascriptNumericLiteralArtifact sub JavascriptSourceArtifact,
-	# Attributes
-	has numberValue,
```

Comment:
Probably a non-issue but do you know why `numberValue` was removed from `JavascriptNumericLiteralArtifact`? I would think that type of artifact would have a number value. If you're not sure that's okay. The tests we have pass. Just curious.

Reply:
Hmm actually yes I have no idea :-(

Reply:
https://github.com/CodeBrig/Arthur/pull/79/commits/cdec6f691597049cd40dcd27256adc20c5aa8343 seems to solve it :)

Reply:
Nice catch :-)